pkg/core/state/notification_event.go

```diff
@@ -24,36 +24,28 @@ type ContractInvocation struct {
 	Hash           util.Uint160     `json:"contract_hash"`
 	Method         string           `json:"method"`
 	Arguments      *stackitem.Array `json:"arguments"`
+	ArgumentsBytes []byte           `json:"arguments_bytes"`
```

Comment:
That's too many details for a public structure used in popular APIs. This should be hidden.